robottelo/vm_capsule.py

```diff
@@ -203,13 +203,13 @@ def _capsule_cleanup(self):
                 # exception has 'return_code=70(Error: host not found)'
                 if exp.return_code == 70:
                     super(CapsuleVirtualMachine, self).destroy()
-                if bz_bug_is_open('1622064'):
-                    logger.warn('Failed to cleanup the host: {0}\n{1}'.format(
-                        self.hostname, exp))
-                else:
-                    logger.error('Failed to cleanup the host: {0}\n{1}'.format(
-                        self.hostname, exp))
-                    raise
+
+                # BEGIN BZ:1622064
+                logger.warn('Failed to cleanup the host: {0}\n{1}'.format(
+                    self.hostname, exp))
+                # when BZ is closed the exception should `raise` here.
```

Comment:
so how exactly I'll find out when the BZ gets closed and I'm supposed to update something here?

Reply:
You will receive an email saying "BZ number 1622064 is now ON_QA, action required on robottelo/vm_capsule.py".

Then you will need to figure out what to do based on the BZ , code context + any good comment that was left on the block. To ensure that we will need to be attentive on reviewing the addition of those blocks.

Reply:
and of course you will need to verify the BZ state before making changes on the code